cli/azd/cmd/up.go

```diff
@@ -23,7 +26,9 @@ func upCmd(rootOptions *internal.GlobalCommandOptions) *cobra.Command {
 				},
 			},
 			&infraCreateAction{
-				rootOptions: rootOptions,
+				// Delay print final output from infra create
+				finalOutputRedirect: &infaCreateFinalOutput,
```

Comment:
This is pretty hacky but works for us now. Will be resolved when we remove composite actions in favor of well-defined `Action` class in #625